IslandLanding/IslandLanding/ViewModel/GamerTagViewModel.cs

```diff
@@ -0,0 +1,31 @@
+﻿using IslandLanding.Views;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Windows.Input;
+using Xamarin.Essentials;
+using Xamarin.Forms;
+
+namespace IslandLanding.ViewModel
+{
+  public class GamerTagViewModel : BaseViewModel
+  {
+    private string _userTag;
+    public string UserTag
+    {
+      get => _userTag;
+      set => SetProperty(ref _userTag, value);
+    }
+    public ICommand SaveCommand { get; set; }
+    public GamerTagViewModel()
+    {
+      SaveCommand = new Command(SaveCommandExcute);
+    }
+
+    private void SaveCommandExcute(object obj)
+    {
+      Preferences.Set("userTag", UserTag);
+      App.Current.MainPage.Navigation.PushModalAsync(new TutorialPage());
```

Comment:
Please don't use PushModalAsync as it doesn't use the right APIs to navigate. We need to use PushAsync. @amiraelsayedbadwy

Reply:
i just use it to test i will fix it in tutorial pull request